source/docs/collections-extending-parent-templates.md

```diff
@@ -89,6 +89,8 @@ If using multiple parent templates, you can specify separate paths in `config.ph
 ```
 <?php
 
+use Illuminate\Support\Str;
```

Comment:
Do we need to use this, since Facades are aliased at the root namespace and this file has no namespace?